docs/source/_ext/scylladb_markdown.py

```diff
@@ -0,0 +1,65 @@
+import os
```

Comment:
is this specific to our project or can this live in the theme?

Reply:
We can move it to the theme and distribute it as a new optional extension. However, I would prefer first to test against this project. Note that not all projects use MD to write their docs.